example/rebar.config.script

```diff
@@ -0,0 +1,6 @@
+%% To avoid specifying a version we add cowboy_swagger in the
+%% _checkouts directory.
+os:cmd("mkdir _checkouts; ln -s $PWD/.. _checkouts/cowboy_swagger").
```

Comment:
Shouldn't this be a pre_commit hook in rebar.config instead of something added to rebar.config.script?

Reply:
I tried an approach with a [shell `pre_hook`](https://www.rebar3.org/v3/docs/configuration#section-shell-hooks), but the command doesn't get executed before the dependencies are resolved. The `rebar.config.script` is evaluated before anything else is done. I don't love this workaround, but having the example outdated was worse IMO.

Reply:
understood.